Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/test_data/pagination_data.json

```diff
@@ -0,0 +1,2900 @@
+                    "integration": false,
+                    "deprecated": false
+                },
+                {
+                    "id": 190,
+                    "type": "content",
+                    "label": "Search Engines and Portals",
+                    "integration": false,
+                    "deprecated": false
+                }
+            ],
+            "verdict": "allowed",
+            "domain": "mtalk.google.com",
+            "timestamp": 1662749723000,
+            "blockedapplications": [],
+            "allowedapplications": [],
+            "identities": [
+                {
+                    "id": 589064228,
+                    "type": {
+                        "id": 9,
+                        "type": "roaming",
+                        "label": "Roaming Computers"
+                    },
+                    "label": "DESKTOP-IIQVPJ7",
+                    "deleted": false
+                }
+            ],
+            "threats": []
+        },
+        {
+            "returncode": 0,
+            "externalip": "1.1.1.1",
+            "allapplications": [
+                {
+                    "id": 997236,
+                    "label": "Windows Diagnostics Data",
+                    "category": {
+                        "id": 44,
+                        "label": "IT Service Management"
+                    }
+                }
+            ],
+            "date": "2022-09-09",
+            "internalip": "1.1.1.1",
+            "time": "18:55:21",
+            "querytype": "AAAA",
+            "policycategories": [],
+            "type": "dns",
+            "categories": [
+                {
+                    "id": 25,
+                    "type": "content",
+                    "label": "Software/Technology",
+                    "integration": false,
+                    "deprecated": true
+                },
+                {
+                    "id": 32,
+                    "type": "content",
+                    "label": "Business Services",
+                    "integration": false,
+                    "deprecated": true
+                },
+                {
+                    "id": 148,
+                    "type": "application",
+                    "label": "Application",
+                    "integration": false,
+                    "deprecated": false
+                },
+                {
+                    "id": 163,
+                    "type": "content",
+                    "label": "Business and Industry",
+                    "integration": false,
+                    "deprecated": false
+                },
+                {
+                    "id": 167,
+                    "type": "content",
+                    "label": "Computers and Internet",
+                    "integration": false,
+                    "deprecated": false
+                }
+            ],
+            "verdict": "allowed",
+            "domain": "ic3.events.data.microsoft.com",
```

Comment:
is this domain a dummy one?

Reply:
@RamboV - pagination data file was removed.  Was it on purpose?

Reply:
@ShacharKidor 
we are not using it now